string target_name
geometry_msgs/Pose[] poses_up
geometry_msgs/Pose[] poses_down
